scopes/envs/envs/envs.docs.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-description: Manages Environments and Environment Services
+description: A composable engine for the standardization of component development environments.
```

Comment:
A composable engine for creating and maintaining development environments.

Reply:
done